gitlab_runner/datadog_checks/gitlab_runner/gitlab_runner.py

```diff
@@ -89,7 +90,7 @@ def _create_gitlab_runner_prometheus_instance(self, instance, init_config):
                 # Defaults that were set when gitlab_runner was based on PrometheusCheck
                 'send_monotonic_counter': instance.get('send_monotonic_counter', False),
                 'health_service_check': instance.get('health_service_check', False),
-                'metadata_metric_name': 'ci_runner_version_info',
+                'metadata_metric_name': 'gitlab_runner_version_info',
```

Comment:
Not sure about this change. It will change the behaviour of the check for very old versions that are still using `ci_runner_version_info` but those versions are not supported anymore

Reply:
I don't think it's possible to provide to value for the `metadata_metric_name` attribute. Is it possible to dynamycally update it?

Reply:
Is it possible to dynamically update the `metadata_metric_name` depending on the gitlab version? Or provide several metrics?

Reply:
Hm I don't think we have a way now, we'd have to update the base check. You could also submit the metadata with `self.set_metadata` like in other checks. In what version did the version metric name change?

Reply:
It was renamed in v11 https://gitlab.com/gitlab-org/gitlab-runner/-/merge_requests/912